### ISSUE TITLE
ZNC gets compiled without charset support

### ISSUE THREAD

**Issue by Mikaela** [Homebrew/homebrew#36982]
On Ubuntu charset support requires `libicu-dev`, but I have no idea what is the equivalent in homebrew if it's even available. It should probably be added as requirement of the ZNC formula.

**Comment by tdsmith** [comment]
Thanks! Does adding `depends_on "icu4c"` do the right thing? What's charset support mean in the context of ZNC?

**Comment by Mikaela** [comment]
> Thanks! Does adding depends_on "icu4c" do the right thing?

I don't know and I don't have access to OS X for some hours.

> What's charset support mean in the context of ZNC?

It adds support for ZNC to read multiple character sets and convert between them for clients that expect to receive some special charset, but server sends some specific one or the other way.

**Comment by Mikaela** [comment]
If you try it, `znc --version` should returnn

```
ZNC - 1.7.0-git-6-a8b9542 - http://znc.in
IPv6: yes, SSL: yes, DNS: threads, charset: yes
```

This Linux shell has version compiled from git, so you will see 1.6.0 instead of something-git-hash.

**Comment by DomT4** [comment]
Fix in #36990.

**Comment by Mikaela** [comment]
Thanks, I can confirm that fix as working :)

```
# znc --version
ZNC - 1.6.0 - http://znc.in
IPv6: yes, SSL: yes, DNS: threads, charset: yes
```
